src/js/components/Heading/README.md

```diff
@@ -22,6 +22,10 @@ The sizing can be further adjusted using the size property.
 2
 3
 4
+1
```

Comment:
what is the `+1` here? Should that be quotes instead?

Reply:
That's what `react-desc` does.